Sources/SwiftParser/Lexer/Cursor.swift

```diff
@@ -1856,7 +1856,7 @@ extension Lexer.Cursor {
 
   /// Returns `true` if the cursor is positioned at `\##(` with `delimiterLength`
   /// pound characters.
-  private func isAtStringInterpolationAnchor(delimiterLength: Int) -> Bool {
+  private func atStringInterpolationAnchor(delimiterLength: Int) -> Bool {
```

Comment:
Oh, I put the previous comment on the wrong line. I meant that we should keep the functions in Cursor.swift. Everything else should be renamed. I’m sorry.

Reply:
Ah, mb, I should've read it more carefully too. Okay, so reverting this change, but renaming SwiftParser/Expressions.swift.